Release.toml

```
@@ -1,4 +1,4 @@
-version = "1.6.0"
+version = "1.7.0"
```

Let's not bump this version tag for now. We bump this when we make a new release.
```suggestion
version = "1.6.0"
```